projects/lib-shared-modules/src/lib/components/search/search.component.html

```diff
@@ -1,6 +1,11 @@
 <div class="search-container flex flex-row p-1 gap-2" >
-    <div class="flex text-center">
-        <mat-icon class="search-icon" matPrefix (click)="onSearch(searchInput.value)">search</mat-icon>
+    <div class="flex text-center flex-grow">
+        <mat-icon class="search-icon my-1 ml-2 cursor-pointer" matPrefix (click)="onSearch(searchInput.value)">search</mat-icon>
     </div>
-    <input matInput #searchInput [formControl]="searchControl" [attr.maxlength]="maxLength"  (keyup.enter)="onSearch(searchInput.value)" placeholder="Search" >
+    <input type="text" matInput #searchInput [formControl]="searchControl" [attr.maxlength]="maxLength"  (keyup.enter)="onSearch(searchInput.value)" placeholder="Search" class="w-full pl-1 pr-8 py-1" > 
```

Comment:
https://www.w3schools.com/tags/att_input_type_search.asp can you please check this and try implement this. please check the impact if any

Reply:
this is not working properly as the cross icon on search bar is  hides while clicking outside the searchbar.